README.md

```diff
@@ -130,8 +130,8 @@ strategies:
 
 ### RemoveDuplicates
 
-This strategy makes sure that there is only one pod associated with a Replica Set (RS),
-Replication Controller (RC), Deployment, or Job running on the same node. If there are more,
+This strategy makes sure that there is only one pod associated with a ReplicaSet (RS),
+ReplicationController (RC), StatefulSet, Deployment, or Job running on the same node. If there are more,
```

Comment:
We might drop mention of Deployment or stress a Deployment creates a ReplicaSet which then creates specified number of pods.

Reply:
I removed Deployment. I amended the commit and force pushed.